adafruit_tcs34725.py

```diff
@@ -174,7 +191,7 @@ def gain(self, val):
 
     @property
     def interrupt(self):
-        """Get and clear the interrupt of the sensor.  Returns a bool that's
+        """Return and clear the interrupt of the sensor.  Returns a bool that's
```

Comment:
Delete this line entirely. When finished, should read `True if the interrupt is set. Can be set to False (and only False) to clear the interrupt.`